src/Surfnet/StepupMiddleware/CommandHandlingBundle/EventListener/CommandProcessingExceptionListener.php

```diff
@@ -18,13 +18,13 @@
 
 namespace Surfnet\StepupMiddleware\CommandHandlingBundle\EventListener;
 
-use Surfnet\StepupMiddleware\CommandHandlingBundle\Pipeline\Exception\InvalidCommandException;
+use Surfnet\StepupMiddleware\CommandHandlingBundle\Pipeline\Exception\ProcessingAbortedException;
 use Symfony\Component\EventDispatcher\EventSubscriberInterface;
 use Symfony\Component\HttpFoundation\JsonResponse;
 use Symfony\Component\HttpKernel\Event\GetResponseForExceptionEvent;
 use Symfony\Component\HttpKernel\KernelEvents;
 
-class InvalidCommandExceptionListener implements EventSubscriberInterface
+class CommandProcessingExceptionListener implements EventSubscriberInterface
```

Comment:
As an aside: what would you say about making a tech debt issue that allows for catching this specific exception at the controller level? Seems like something the controller would do. Would make it more apparent that this exists and handled...